lib/reporters/base.js

```diff
@@ -6,15 +6,24 @@
  * Module dependencies.
  */
 
-var tty = require('tty');
 var diff = require('diff');
 var milliseconds = require('ms');
 var utils = require('../utils');
-var supportsColor = utils.isBrowser() ? null : require('supports-color');
+var supportsColor = require('supports-color');
 var constants = require('../runner').constants;
 var EVENT_TEST_PASS = constants.EVENT_TEST_PASS;
 var EVENT_TEST_FAIL = constants.EVENT_TEST_FAIL;
 
+var isBrowser = !!process.browser;
```

Comment:
use `utils.isBrowser()`.  this makes testing easier

Reply:
Fixed in 4879b0c0